coss/opensource_clubs/models.py

```diff
@@ -129,3 +129,16 @@ class FAQPage(Page):
     content_panels = Page.content_panels + [
         StreamFieldPanel('faq'),
     ]
+
+
+class ActivitiesPage(Page):
+    """Activities Page for Open Source Clubs."""
+    description = RichTextField(blank=True)
```

Comment:
We can make the whole card a streamfield which we can also use for the collaborators section.

Reply:
Regardless of the streamfield, the design has a photo section too. Let's add this functionality and if no photo is provided we can default to the one listed in the design.